datadog-logs-oci-orm/logs-function/func.py

```diff
@@ -0,0 +1,112 @@
+import io
+import os
+import json
+import logging
+import gzip
+import requests
+
+logger = logging.getLogger(__name__)
+
+DD_SOURCE = "oracle_cloud"  # Adding a source name.
+DD_SERVICE = "OCI Logs"  # Adding a service name.
+DD_TIMEOUT = 10 * 60  # Adding a timeout for the Datadog API call.
+DD_BATCH_SIZE = 1000  # Adding a batch size for the Datadog API call.
```

Comment:
Is there a max size that can be sent to DD logs endpoint? I see that this batch is 1000 logs but I'm curious if we might need to enforce a size in bytes that are sent via the API?

Reply:
I think the max size for the batch size is 1000, as enforced by Datadog. Since, logs are compressed before calling API, I think the byte limit shouldn't be a problem. Here are the limits: https://docs.datadoghq.com/api/latest/logs/#send-logs